spec/rails_helper.rb

```diff
@@ -70,6 +70,10 @@
 ActiveJob::Base.queue_adapter = :test
 
 Capybara.server = :webrick
+Capybara.server_host = Rails.application.routes.url_helpers.root_url
+Capybara.app_host = Rails.application.routes.url_helpers.root_url
+Capybara.asset_host = Rails.application.routes.url_helpers.root_url
+Capybara.server_port = Rails.application.default_url_options[:port]
```

Comment:
I'm not sure all of these are necessary but this ensures we're using the configured host and not the default of `www.example.com`

Reply:
Looks good. I'd rather over-test than under